### DISCUSSION TITLE
Dot Badge

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#308] [Component suggestions]
<img width="480" alt="visuele weergave van de dot badge component" src="https://github.com/user-attachments/assets/329d5145-497e-4422-948f-b1ffa37b6736">

We willen graag meer leren over de Dot Badge component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, GitHub of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1734524100734959)

---

## Naam

Dot badge

## Doel
Stip bij een Link of Button die zonder zichtbaar label de aandacht trekt bij een statuswijziging.

## Gerelateerde componenten

[Data Badge](https://github.com/orgs/nl-design-system/discussions/307), [Number Badge](https://github.com/orgs/nl-design-system/discussions/214), [Status Badge](https://github.com/orgs/nl-design-system/discussions/290)

## Onderzoek

[Rapport Generieke Services 2023 - Gemeente Den Haag (PDF, 9 MB)](https://github.com/user-attachments/files/16993296/Rapport.generieke.services.23032023.def.pdf)
Uit dit gebruikersonderzoek blijkt dat de ‘blauwe bolletjes’ naast een bericht vaak verkeerd werden geïnterpreteerd. Gebruikers dachten dat deze een belangrijk bericht aanduidden, terwijl ze in werkelijkheid ongeopende berichten markeerden. De aanduiding met het blauwe bolletje bij ‘Mijn berichten’ in de zijnavigatie werd daarentegen wel correct begrepen.

[Onderzoek Generieke Services - Miro](https://miro.com/app/board/uXjVP0OE2fg=/?share_link_id=511269957959)
Miro board dat is gebruikt tijdens het gebruikersonderzoek waarin inzichten te vinden zijn over de Dot Badge.

[Badges - Material Design](https://m3.material.io/components/badges/overview)
Material Design gebruikt de 'Small badge' om aan te geven dat er een melding ongelezen is en wordt het meest gebruikt binnen andere componenten, zoals de navigation bar, navigation rail, top app bars, and tabs. Een screenreader kondigt "nieuwe melding" aan.

[Badge notifications: why everyone wants to get rid of them - Medium](https://uxdesign.cc/badge-notifications-why-everyone-wants-to-get-rid-of-them-decedb28b46d)
In dit artikel wordt aangegeven dat notificatiebadges veel aandacht trekken door hun associatie met urgentie en gevaar, wat gebruikers motiveert om actie te ondernemen. Ze activeren cognitieve biases zoals salience (focus op opvallende zaken) en urgency (illusie van urgentie), waardoor apps met badges sneller worden geopend. Tegelijkertijd kunnen deze badges overweldigend en stressvol zijn, wat gebruikers ertoe aanzet om manieren te zoeken om ze uit te schakelen.

**Comment by Rozerinay** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/react-data-display-dot-indicator--docs)
- [Figma - Huidig](https://www.figma.com/design/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?node-id=1540-4477&t=FHNfoNl8RtCbcTFM-11)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/DotIndicator)

---

### Screenshots / Varianten

<img width="195" alt="image" src="https://github.com/user-attachments/assets/a9a4371d-5cc7-4009-a9ee-bd557b252685" />

---

### Notities

- Onder de naam: Dot Indicator.
- Is een `<span>`.
- Positie wordt bepaald met de property `position`.
- Styling: rode stip met een witte border.
- Richtlijnen beschikbaar.

**Comment by jeffreylauwers** [comment]
## Rijkshuisstijl Community

- [Storybook](https://rijkshuisstijl-community.vercel.app/?path=/docs/rhc-dot-badge--docs)
- [Figma](https://www.figma.com/design/txFX5MGRf4O904dtIFcGTF/NLDS---Rijkshuisstijl---Bibliotheek?node-id=17541-128)
- [GitHub](https://github.com/nl-design-system/rijkshuisstijl-community/tree/main/packages/components-css/src/dot-badge)

---

### Screenshots / Varianten

![screenshot storybook van dotbadge als rode stip](https://github.com/user-attachments/assets/97f2350d-9a8f-4275-ad15-4820843dea73)


---

### Notities

- Styling: rode stip
- Via `aria-label` is de tekst instelbaar. In Storybook staat nu "Nieuw Bericht".

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Help Wanted ✨


Help je mee hem door de Community stappen te krijgen?
[Dot Badge op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=98)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Dot Badge op het Candidate bord](https://github.com/orgs/nl-design-system/projects/80/views/1?filterQuery=Dot+Badge)
